DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/TutorialViewModelTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.ViewModels.LearningMenu
+{
+    using DigitalLearningSolutions.Web.Tests.TestHelpers;
+    using DigitalLearningSolutions.Web.ViewModels.LearningMenu;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    public class TutorialViewModelTests

```

Comment:
Another thing, I believe you have many helper methods in the view model which you're not testing here? 🙂

Reply:
Yes, sorry I forgot about those bits. Added more tests in 5dda1cc 🙂